roborock/data/__init__.py

```diff
@@ -0,0 +1,9 @@
+"""This module is meant to hold dataclasses and codemappings for various devices and protocols."""
+
+from .b01_q7 import *
```

Comment:
These will all fit in the same namespace w/o any overlap? I'm a little unsure when we should import from `.data` vs importing from subdirectories.

Reply:
Sounds good to me.